cmd/git-sync/main.go

```diff
@@ -217,8 +219,10 @@ func main() {
 		os.Exit(1)
 	}
 
+	ctx, cancel := context.WithTimeout(context.Background(), initTimeout)
```

Comment:
Can you please comment the scope of this (just initialization, not clone, etc) and why 30s is "enough" ?

Reply:
Done